flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/read/HiveTableInputFormat.java

```diff
@@ -103,17 +105,16 @@ public HiveTableInputFormat(
 		this.jobConf = new JobConf(jobConf);
 		int rowArity = catalogTable.getSchema().getFieldCount();
 		selectedFields = projectedFields != null ? projectedFields : IntStream.range(0, rowArity).toArray();
+		useMapRedReader = GlobalConfiguration.loadConfiguration().getBoolean(HiveOptions.TABLE_EXEC_HIVE_FALLBACK_MAPRED_READER);
```

Comment:
You can only pass `useMapRedReader` here.
We already have `GlobalConfiguration.loadConfiguration()` in `tableSource.getDataStream`. We can just get `useMapRedReader` and pass it to input format. It is good to reduce this unhappy codes.

Reply:
Done